test/appium/tests/atomic/chats/test_public.py

```diff
@@ -328,6 +329,304 @@ def test_public_chat_open_using_deep_link(self):
             self.driver.fail("Public chat '%s' is not opened" % chat_name)
 
 
+@pytest.mark.xdist_group(name="messaging_1")
+class TestMessaging(MultipleSharedDeviceTestCase):
+
+    @classmethod
+    def setup_class(cls):
+        cls.drivers, cls.loop = create_shared_drivers(2)
+        cls.device_1, cls.device_2 = SignInView(cls.drivers[0]), SignInView(cls.drivers[1])
+        cls.home_1, cls.home_2 = cls.device_1.create_user(enable_notifications=True), cls.device_2.create_user()
+        cls.public_key_1, cls.default_username_1 = cls.home_1.get_public_key_and_username(return_username=True)
+        cls.public_key_2, cls.default_username_2 = cls.home_2.get_public_key_and_username(return_username=True)
+        profile_2 = cls.home_2.get_profile_view()
+        profile_2.switch_network()
+        [home.home_button.click() for home in (cls.home_1, cls.home_2)]
+        cls.chat_2 = cls.home_2.add_contact(cls.public_key_1)
+        cls.chat_2.chat_message_input.send_keys('test message')
```

Comment:
send_message. What is the reason for sending this message?

Reply:
This piece of code was copied from original test case @marks.testrail_id(5425). I've checked this test case and didn't find any reason why we are sending this "test message". I guess I can remove this code unless there is still any reason which I cannot realise. Please, take a look at this @marks.testrail_id(5425) test case. If you approve that this piece of code is unnecessary I will remove it.

Reply:
you can send a message, but better to work with them afterward, as I mentioned https://github.com/status-im/status-react/pull/13163/files#r825764748